openwisp_notifications/static/openwisp-notifications/js/object-notifications.js

```diff
@@ -15,9 +15,9 @@
 
 function getObjectNotificationComponent() {
     return `
-    <div class="ow-object-notification-container">
+    <li class="ow-object-notification-container">
         <button id="ow-object-notify" class="ow-object-notify button" title="You are receiving notifications for this object.">
-            <div class="ow-icon ow-object-notify-bell"></div>
+            <i class="ow-icon ow-object-notify-bell"></i>
```

Comment:
I think `<span>` is more appropriate then `<i>` here! 😄

Reply:
I thought we are using ``<i>`` for icons in other places so I changed it

Reply:
`<i>` is actually wrong, we should change at other places, but it's not a priority.